### PR TITLE
fix: added _size to the report generator for problem responses

### PR DESCRIPTION
**Pull request by jnlapierre**
**TL;DR -** A new parameter was added to the dict and caused the celery task to fail when generating the report

JIRA: [AU-522](https://openedx.atlassian.net/browse/AU-522)

**What changed?**

Added the _size value to be processed

**Developer Checklist**
- [ ] Bumped version number in [setup.py](https://github.com/edx/edx-ora2/blob/a62e81a9b0d89223476967ec3c27f3557a850735/setup.py#L39) and [package.json](https://github.com/edx/edx-ora2/blob/a62e81a9b0d89223476967ec3c27f3557a850735/package.json#L3)

**Testing Instructions**

Working on testing it locally in DevStack.

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:

- [ ] I've done a visual code review
- [ ] I've tested the new functionality

FYI: @openedx/content-aurora

### CODE REVIEW

**Review by mattcarter — APPROVED** [review]
LGTM 👍
